lib/src/builder/generation/impl_generation.dart

```diff
@@ -570,20 +574,21 @@ class ImplGenerator {
     }
 
     final consumerClassName = _classNameFromNode(node);
-    final accessorsMixinName = '\$$consumerClassName';
+    final accessorsMixinName = consumerClassName.startsWith(generatedPrefix) ?
```

Comment:
#nit Should this be `companionPrefix`? (I know it's not declared in this file, but it would be nice to be consistent between this and impl_generation.dart)

Reply:
How about `buildTargetPrefix`, since anything prefixed with `_$` will have an analogous built class. I'm not in love with it, since it overloads the "build target" terminology, but I can't think of anything better atm 🤷‍♂️ 

Thoughts?

Reply:
What about something like `privateSourcePrefix`? I was trying to find a precedent for referring to these names, and it looks like built_value uses the term "source" https://github.com/google/built_value.dart/tree/master/built_value_generator/lib/src

Other "companion" alternative name ideas:
- input/buildInput
- template/buildTemplate
- boilerplate
- precursor

Reply:
I like `privateSourcePrefix`